trojsten/events/templates/trojsten/events/participants_and_organizers_list.html

```diff
@@ -23,13 +23,17 @@ <h1>Zoznam účastníkov</h1>
         <td>{{ participant.user.get_full_name }}</td>
         <td>{{ participant.year_at_event | school_year }}</td>
         <td>
-        {% if participant.user.school.has_abbreviation %}
-            <abbr title="{{ participant.user.school.verbose_name }}"
-                  data-toggle="tooltip" data-placement="top">
-                {{ participant.user.school.abbreviation }}
-            </abbr>
+        {% if participant.user.school %}
+            {% if participant.user.school.has_abbreviation %}
+                <abbr title="{{ participant.user.school.verbose_name }}"
+                      data-toggle="tooltip" data-placement="top">
+                    {{ participant.user.school.abbreviation }}
+                </abbr>
+            {% else %}
+                {{ participant.user.school.verbose_name }}
```

Comment:
Hoci momentálne majú všetky školy skratku (prečo to vlastne nie je povinné aj spolu s menom?),
ak by nejaká nemala, dĺžka 100 je dosť veľa a verbose name tak môže asi rozbíjať tabuľku.

Skús sa pozrieť na to, ako veľmi by to bolo zlé ak by tam bol dlhší názov
a navrhoval by som "automaticky generovať" skratku ak skratka nie je definovaná: spraviť škole funkciu, ktorá z názvu zoberie napr prvých 10 znakov a potom doplní tri bodky ... alebo možno spraviť abbreviation ako povinné? @mhozza @dodo42

Reply:
Mozme spravit abbr povinnu. Moze to byt v zvlast PR.

Reply:
Ako to tak pozerám, 600 / 2400 škôl nemá skratku, takže asi ju nechcem robiť povinnou.
Vyzerá to tak ale tak, že z týchto škôl nemáme riešiteľov, alebo ak sa takáto škola vyskytne, vedúci jej vytvorí skratku...

Takže asi pass

Reply:
co takto automaticky generovana skratka prve pismeno z kazdeho slova nazvu? koniec koncov dost skol ma aj realne skratku generovanu tymto sposobom

```
@property
def abbrev(self):
    if self.abbreviation:
        return self.abbreviation
    return "".join(map(lambda x: x[0], self.verbose_name.split()))
```